src/signals/settings/categories/Detail/components/CategoryForm/CategoryForm.tsx

```diff
@@ -95,6 +97,27 @@ const CategoryForm: FunctionComponent<CategoryFormProps> = ({
     [data]
   )
 
+  useEffect(() => {
+    if (data?.is_public_accessible) {
+      setIsPublicAccessible(data.is_public_accessible)
+    }
+    if (data?.public_name) {
+      setPublicName(data.public_name)
+    }
```

Comment:
Same here; the state value can be set on component instantiation.

Reply:
done